css/properties/transform-origin.json

```diff
@@ -174,10 +174,10 @@
             },
             "support": {
               "chrome": {
-                "version_added": true
+                "version_added": "12"
```

Comment:
Looking good in this PR!  Just got a quick question about this line -- from comparing WebKit versions, Safari 4.1 would be Chrome 6.  May I ask where "12" comes from?

Reply:
[`transform`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform#Browser_compatibility) has a 3D feature which reports version 12 and the [version 12 blog post](https://chromereleases.googleblog.com/2011/06/chrome-stable-release.html) also mentions the availability of "3D CSS". It make a lot of sense to me to record that Chrome supported a Z offset for `transform-origin` before other 3D transforms (what would it even mean?).

Reply:
Gotcha -- alrighty then, thanks for clarification, looking good to me!